openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/HealthCheck.spec.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ *  Copyright 2024 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { GlobalSettingOptions } from '../../constants/settings.constant';
+
+describe('Health Check for Openmetadata', () => {
+  before(() => {
+    cy.login();
+  });
+
+  it('All 5 checks should be successful', () => {
+    cy.settingClick(GlobalSettingOptions.OM_HEALTH);
```

Comment:
I think we have to inspect the status API here so that we can have the latest response and then only we check the status.

Reply:
+1